app/utils.py

```diff
@@ -273,21 +274,26 @@ def validate_uk_mobile_phone_number(number, locale):
 class ProcessName:
 
     @staticmethod
-    def validate_name(request, data, display_region):
+    def validate_name(request, data, display_region, child=False):
 
         name_valid = True
         form_first_name = data.get('name_first_name')
         form_last_name = data.get('name_last_name')
 
```

Comment:
This could all be a bit more succinct if we use a variable to remove the extra levels of if / else:
```
whose_name = "child's " if child else ""
...
  flash(request, {'text': "Enter your " + whose_name + "first name", # etc
...
 flash(request, {'text': "Enter your " + whose_name + "last name", # etc
...


```

Reply:
Agreed that as the text could yet diverge further to leave as is, and look to compact if possible when text is comfirmed